data-es7.js

```diff
@@ -1190,6 +1190,63 @@ exports.tests = [
   res: {
     babel:       true,
   }
+},
+{
+  name: 'Observable',
+  category: 'proposal',
+  significance: 'large',
+  link: 'https://github.com/zenparsing/es-observable',
+  'subtests': {
+    'basic support': {
+      exec: function () {/*
+        return typeof Observable !== 'undefined';
+      */}
+    },
+    'Symbol.observable well known symbol': {
+      exec: function () {/*
+        return typeof Symbol.observable === 'symbol';
+      */}
+    },
+    'Observable.prototype.subscribe': {
+      exec: function () {/*
+        return 'subscribe' in Observable.prototype;
+      */}
+    },
+    'Observable.prototype.forEach': {
+      exec: function () {/*        
+        var o = new Observable();
```

Comment:
Yes - `If IsCallable(subscriber) is false, throw a TypeError exception.`, step 2 in https://github.com/zenparsing/es-observable#observablesubscriber

We should also verify step 1 - that it throws a `TypeError` when it's not invoked with `new`.